app/models/room.rb

```diff
@@ -0,0 +1,47 @@
+
+class Room < ActiveRecord::Base
+  belongs_to :week
+  belongs_to :user
+  has_many :rooms_learning_objects
+  has_many :user_to_lo_relation
+  has_many :learning_objects, through: :rooms_learning_objects
+
+  # Vrati learning objecty, ktore v danom pokuse a miestnosti este neboli videne
+  def get_dont_visited
```

Comment:
V nazvoch metod by som odporucal pouzivat skor neurcity zapor: `get_not_visited`.

Zaroven, v tomto pripade by mohol nazov metody navadzat, ze pozadujeme nie _vzdelavacie objekty_, ale nenavstivene _miestnosti_. To sice moze byt ramci vyznamu nieco podobne, ale cisto z programatorskeho hladiska tato metoda nevracia zoznam instancii triedy `Room`, ale triedy `LearningObject`. Preto by som to skor nazval `get_not_visited_los`.